vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -202,29 +202,42 @@ An `env.json` file to configure the application URLs and client credentials must
 Instructions to export a bundle, including how to configure `env.json`, can be found in [this tutorial](../../tutorials/ecr/export-bundle-from-application.md).
 
 ## Profile Management
-To manage and switch between different configurations use `ent profile`. To switch between different Entando Applications, even if they are in different clusters, `ent profile` is instructed to use Kubernetes contexts, kubeconfig files, and/or custom commands (refer to `ent profile first-use-readme`).
+To manage and switch between different `ent` configurations, use `ent profile`. 
 
-1. Create a new profile
+To switch between different Entando Applications, even if they are in different clusters, `ent profile` uses Kubernetes contexts, kubeconfig files, and/or custom commands (refer to [`ent profile first-use-readme`](https://github.com/entando/entando-cli/blob/develop/bin/mod/ent-profile)).
```

Comment:
Let's not point to the code itself since it could move. I think referring to the command is fine and it will show the actual readme for the current version whereas if you go to the code you could be looking at the wrong branch or for a different version altogether.